system/netdata.service.in

```diff
@@ -8,7 +8,7 @@ WorkingDirectory=/tmp
 User=root
 Group=root
```

Comment:
should be:
```
Type=simple
User=@NETDATA_USER@
```

and you should add to configure.ac subst of NETDATA_USER with ${with_user}, also fix the openrc script with this as default.